### ISSUE TITLE
Background and foreground colors do not have a sufficient contrast ratio

### ISSUE THREAD

**Issue by spaceninja** [MLTSHP/mltshp-patterns#257]
Low-contrast text is difficult or impossible for many users to read. [Learn more](https://web.dev/color-contrast/).

(There are too many failing elements to list. You can check using Storybook's accessibility panel or by running Lighthouse on the production site)
